microsoft/botframework-solutions
# [Enterprise Bot Template][TS] Review and sync deployment script.

> This pull-request just modify the deployment script in TypeScript to sync it with the C# version. However, the code-behavior is still different between both version so we created #536 to keep track that issue.

> We'll create another pull-request to solve the behavior gap.

## Description
The structure of the Deployment Scripts between C# and TypeScript now contain the same scripts. For this purpose, we changed the following things:

- Change the project structure on Typescript. 
- Add `ES`, `FR`, `IT`, `ZH`, DE folders inside deployment script folder with their respective files.
- Change `EN` location and update the files that contains.
- Delete the `msbotclone` folder.
- Rename the files of each culture, except those with the name `faq.qna` and `bot.recipe`.

![image](https://user-images.githubusercontent.com/42946572/50990971-4db72480-14f2-11e9-9c70-ccfde1d9701f.png)


## Related Issue

[Enterprise Bot Template] Sync the structure of Deployment Script between C# and TypeScript #530

## Testing Steps

1. Open `AI/templates/Enterprise-Template/src/typescript/deploymentScripts` 
and `AI/templates/Enterprise-Template/src/csharp/enterprisebotsample/Deploymentscripts`.
2. Check the structure between them.

## Checklist

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant

If you have updated responses or `.lu` files:
